lib/messages/pull-request.js

```diff
@@ -106,32 +124,24 @@ class PullRequest extends AbstractIssue {
       };
     }
 
-    const attachments = [
+    let attachments = [
```

Comment:
In JavaScript `const` doesn't mean _immutable_, it just means that the variable cannot be reassigned. So you can keep using `const` here.